File: hw4/hw4.FS/Calculator.fs
module hw4.FS.Calculator

        let Calculate (val1:int) operation (val2:int) =
            if (operation = "/" && val2 = 0) then
                raise (System.DivideByZeroException())
            match operation with
            | "+" -> val1 + val2
            | "-" -> val1 - val2
            | "*" -> val1 * val2
            | "/" -> val1 / val2
            | _ -> 0

File: hw8/hw8/Services/Calculator.cs
using System.Globalization;

namespace hw8.Services
{
    public class Calculator : ICalculateService
    {
        public string Calculate(int val1, string operation, int val2)
        {
            var invC = CultureInfo.InvariantCulture;
            var result = 0;
            result = operation switch
            {
                "plus" => val1 + val2,
                "minus" => val1 - val2,
                "multiply" => val1 * val2,
                "divide" => val1 / val2,
                _ => result
            };
            return result.ToString(invC);
        }
    }
}